scripts/postprocessing/coverage_analysis/coverage_Analyzer.py

```diff
@@ -0,0 +1,624 @@
+
+import sys, os, re, subprocess
+import math
+from tf.transformations import *
+import timeit
+
+""" Determines if a given pose or list of poses in a file is within an Astrobee volume
+    based on a database of known poses and their corresponding ML features
+"""
+
+class Coverage_Analyzer:
+
+  def __init__(self):
+
+     self.grid_size = 0.30 #0.15 #0.30 # meters
+     self.max_pose_error = ((self.grid_size / 2.0)**2) *3   # Max squared error (m^3)   0.0918 0.3675
+     self.max_quat_error = 0.52 #0.2616 # Max relative rotation between two quaternions (in radians equiv to 15 deg) 0.52
+     self.matchCounter = 0
+     print("Grid size: {:.3f}, Max Pose Error: {:.3f}".format(self.grid_size, self.max_pose_error))
+
+     # Constants in meters based on JEM keepin dimensions as defined in
+     # https://babelfish.arc.nasa.gov/bitbucket/projects/ASTROBEE/repos/astrobee_ops/browse/gds/ControlStationConfig/IssWorld-Default/keepins/keepin.json
+     self.xmin_bay16 =  11.99  # Lower FWD corner x coordinate at the entry node, keepin zone from bay 1 to 6
```

Comment:
Would be better to load these from a text file, or at least have an array of points at the top of the file you pass to the class.